src/platform/opengl/gles2.c

```diff
@@ -17,6 +17,10 @@ mLOG_DEFINE_CATEGORY(OPENGL, "OpenGL", "video.ogl");
 
 #define MAX_PASSES 8
 
+#if defined(USE_EPOXY) || defined(BUILD_GL) || defined(USE_GLES3)
```

Comment:
This file shouldn't even be being built if only `BUILD_GL` is set, so this is again wrong. VAO usage is (attempted) to be detected at runtime.